test/integration/smoke/test_deploy_vm_root_resize.py

```diff
@@ -114,7 +114,13 @@ def test_00_deploy_vm_root_resize(self):
         # 2. root disk has new size per listVolumes
         # 3. Rejects non-supported hypervisor types
         """
-        if(self.hypervisor.lower() == 'kvm'):
+        full_clone_config = Configurations.list(self.apiclient,
+                            name="vmware.create.full.clone")[0].value
+        if full_clone_config == 'false' and self.hypervisor.lower() == 'vmware':
+            self.skipTest("root disk resize is not supported "+
+                "when vmware.create.full.clone is %s" % full_clone_config)
+
```

Comment:
thanks for  your comment.Sure will add  the logic  to test_deploy_vm_resize.py as well .